### PR TITLE
Home page buttons should be configurable

### PR DESCRIPTION
**Pull request by brandones**
Opening as a draft because it still needs tests. But I would like feedback on the config API it defines.

### CODE REVIEW

File: src/root.component.tsx [hunk 1]
```diff
@@ -8,6 +8,76 @@ defineConfigSchema("@openmrs/esm-home", {
   buttons: {
     enabled: {
       default: false
```
Comment (jonathandick) [inline src/root.component.tsx]: I think set this to true for now

File: src/root.component.tsx [hunk 2]
```diff
@@ -8,6 +8,76 @@ defineConfigSchema("@openmrs/esm-home", {
...
+          link: {
+            spa: false,
+            url:
+              "/openmrs/registrationapp/registerPatient.page?appId=referenceapplication.registrationapp.registerPatient"
+          }
+        },
+        {
+          label: "Capture Vitals",
+          link: {
+            spa: false,
```
Comment (fatmali) [inline src/root.component.tsx]: why does `spa` default to `false`? don't we already have a mechanism to capture the vitals?

File: src/root.component.tsx (hunk 1 above)
Comment (brandones) [reply src/root.component.tsx]: Per my [versioning recommendation](https://github.com/openmrs/openmrs-rfc-frontend/pull/22/files), I'd like to release a minor version with it disabled, and then turn it on in the next major version. Does that seem reasonable?

File: src/root.component.tsx (hunk 2 above)
Comment (brandones) [reply src/root.component.tsx]: This is a specific "app" for rapidly collecting vitals from lots of different patients (e.g. that would be used by a nurse doing triage). It doesn't yet have a SPA equivalent.

Also, I don't really expect anyone to use the default home button configuration. I think its purpose should be seen as showing the implementer what capabilities are available. I think we should feel pretty free to add stuff, as new MFEs get developed. (My mind is open to being changed on all of this).

**Comment by brandones** [comment]
@jonathandick or @fatmali ready for re-review. Also @jonathandick if you want to talk about what things are named, let's do that soon.

File: src/root.component.tsx (hunk 2 above)
Comment (fatmali) [reply src/root.component.tsx]: sounds good 👍

**Pull request merged**